factcast-store-pgsql/src/test/java/org/factcast/store/pgsql/registry/validation/FactValidatorTest.java

```diff
@@ -76,7 +77,7 @@ public void testFailsToValidateIfValidatableButMissingSchema() throws Exception
         when(props.isValidationEnabled()).thenReturn(true);
         when(props.isAllowUnvalidatedPublish()).thenReturn(false);
         SchemaRegistry sr = mock(SchemaRegistry.class);
-        when(sr.get(Mockito.any())).thenReturn(Optional.empty());
+        when(sr.get(Mockito.any(SchemaKey.class))).thenReturn(Optional.empty());
```

Comment:
Mockito.* is already imported. maybe check IDE setting for fav. static imports?

Reply:
I only fixed what was there before :)

Reply:
LOL, look, who's IDE setup is weird :older_man: 
i'll unify that.